mods/page_revisions.js

```diff
@@ -255,8 +360,9 @@ PRS.prototype.fetchAndStoreMWRevision = function(restbase, req) {
             redirect: redirect
         };
 
+        var actions = [
         // Check if the same revision is already in storage
-        return restbase.get({
+        restbase.get({
```

Comment:
Indentation: this whole block is part of the `actions` array, so it should be indented properly.